const.py

```diff
@@ -13,3 +13,22 @@
 SUMMARY_PII_ENTITY_KEY = "pii_entity"
 SUMMARY_SAMPLES_KEY = "samples"
 SUMMARY_RATIONALES_KEY = "rationales"
+
+# This must be the same as `SensitiveDataClass` defined in data classification service proto
+SENSITIVE_DATA_CLASS_MAP = {
+    "sensitive_data_class_unspecified": 0,
```

Comment:
I think you can actually just pass the uppercase enum name directly to the service, no need to translate to numbers. That's how I invoked the endpoints with curl at least

Reply:
yeah that seems to work!